### PR TITLE
feat(app): Add temporary admin page for sales

### PR DESCRIPTION
**Pull request by ernum**

### CODE REVIEW

File: src/server/api/routers/exhibitor.ts [hunk 1]
```diff
@@ -438,4 +438,26 @@ export const exhibitorRouter = createTRPCRouter({
         },
       });
     }),
+  getExhibitors: publicProcedure.query(async ({ ctx }) => {
```
Comment (mathiasmagnusson) [inline src/server/api/routers/exhibitor.ts:441]: Having a username/password is of no use if this is public

File: src/shared/Classes.ts [hunk 2]
```diff
@@ -100,3 +100,52 @@ export class Package {
 
   getPackage(t: Locale, exhibitorPackage: string) {}
 }
+
+export class Exhibitor {
```
Comment (mathiasmagnusson) [inline src/shared/Classes.ts:104]: Not that important but what is the use of this class? If we just return some shape from the query for exhibitors, typescript knows what to do

**Review by mathiasmagnusson — CHANGES_REQUESTED** [review]

File: src/shared/Classes.ts (hunk 2 above)
Comment (ernum) [reply src/shared/Classes.ts:104]: I thought so too, but TRPC seems to have an issue with it (see image). 
Also, the logos had to be converted to `base64` server-side due to what seems to be a bug.

image

Comment (mathiasmagnusson) [reply src/shared/Classes.ts:104]: I think that is because the logos are just binary data, which can be solved by converting to base64 as you do now.

**Review by mathiasmagnusson — APPROVED** [review]

**Pull request merged**